CHANGELOG.md

```diff
@@ -32,7 +32,8 @@ Mapbox welcomes participation and contributions from everyone.
 ## Dependencies
 * Update gl-native to v10.11.0-beta.1 and common to v23.3.0-beta.1. ([1934](https://github.com/mapbox/mapbox-maps-android/pull/1934))
 
-
+## Known issues
```

Comment:
Nit : should be also added to https://github.com/mapbox/mapbox-maps-android/releases/tag/v10.11.0-beta.1

Reply:
Yes, will do once we agree about the text here.

Reply:
Done